pwndbg/commands/__init__.py

```diff
@@ -683,82 +683,81 @@ def HexOrAddressExpr(s: str) -> int:
 
 def load_commands() -> None:
     # pylint: disable=import-outside-toplevel
-    import pwndbg.dbg
-
-    if pwndbg.dbg.is_gdblib_available():
```

Comment:
@Aplet123 this needs to stay, that's part of making Pwndbg debugger-agnostic and making it available for LLDB.

Can you rebase this PR over the latest dev and fix conflicts?